Tasks/HelmInstallerV1/make.json

```diff
@@ -10,7 +10,8 @@
     "rm": [
         {
             "items": [
-                "node_modules/kubernetes-common/node_modules/vsts-task-lib"
+                "node_modules/kubernetes-common/node_modules/vsts-task-lib",
+                "node_modules/kubernetes-common-v2/node_modules/azure-pipelines-task-lib"
```

Comment:
The first one won't be needed

Reply:
Yes but I kept it there so that ppl building this would clear off their old dependencies as well.